code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -624,6 +619,37 @@
 	var/footstep = 1
 	var/datum/component/mobhook
 
+/obj/item/clothing/suit/space/hardsuit/ancient/mason
+	name = "M.A.S.O.N RIG"
+	desc = "The Multy Advanced Sencer Of Networks RIG type hardsuit. With shining plates to protect against not only rads storms, SM meltdowns, acid proof coating this hulking hardsuit has huds built into its visors.\
+			The scanners allow for at a glace huds on anything a engineer would need. Sadly most of its armor makes the suit bulky and unable to sustain much harm from any weapon other then a bomb. This RIG can hold bags and toolboxes on its external shells.\
+			M.A.S.O.N Rigs are also made to withstand lava as well as ash storms down on lava land or other planets that are hotter then normal with its own environmental hazards. \
+			Only few of these are made to do the shear costs of them for use. Armord as it is, this suit will NOT save you from a nuke, massive bomb, or other larger explosions."
+	icon_state = "hardsuit-ancient"
+	item_state = "anc_hardsuit"
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 5, "energy" = 500, "bomb" = 500, "bio" = 500, "rad" = 500, "fire" = 500, "acid" = 500)
```

Comment:
Dont be tricked, the 500 doesn't prevent death

Reply:
trilby this is a percentage it doesn't go above 100..

Reply:
Yep, intended. Its to stop suit debuffs or other modifiers